.env.j2

```diff
@@ -0,0 +1,13 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Change the following values for local development.
+SECRET_KEY={{ secret_key }}
+
+TRELLO_TOKEN={{ trello_token }}
+TRELLO_API_KEY={{ trello_key }}
+TRELLO_BOARD_ID=63dd25f4eaaf087231fa1c44
```

Comment:
Agree with you the hardcoded ids aren't ideal, but also agree with you that they're fine in this exercise - this comment is mostly to say thanks for leaving a message on the PR because that sort of context is really useful 😄